<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAdvanceEntry</class>
 <widget class="QDialog" name="DlgAdvanceEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance entry</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="0" colspan="6">
      <widget class="QGroupBox" name="grCard">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Card details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="EQLineEdit" name="leCardCode">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="Field" stdset="0">
           <string>f_card</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="EQLineEdit" name="leCardName">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="13" column="1" colspan="5">
      <widget class="EQTextEdit" name="teRemarks">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="Field" stdset="0">
        <string>f_remarks</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2" colspan="2">
      <widget class="EQLineEdit" name="leAmount">
       <property name="Field" stdset="0">
        <string>f_amount</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="6">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnNew">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPrint">
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="1" colspan="5">
      <widget class="EQLineEdit" name="leBalance">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="EQLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="Field" stdset="0">
        <string>f_canceled</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Reservation #</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Payment mode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="EQLineEdit" name="leVaucher"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Invoice</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="5">
      <widget class="EQLineEdit" name="leRoom">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Room</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Voucher</string>
       </property>
      </widget>
     </item>
     <item row="14" column="5">
      <widget class="QPushButton" name="btnPrintTax">
       <property name="text">
        <string>Print tax</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1" colspan="2">
      <widget class="EQLineEdit" name="leTax">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="EQLineEdit" name="leInvoice">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>TAX</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="5">
      <widget class="EQLineEdit" name="leReserveId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_reservation</string>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="EQDateEdit" name="deDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="Field" stdset="0">
        <string>f_date</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="4">
      <widget class="EQLineEdit" name="lePaymentName">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="5">
      <widget class="EQLineEdit" name="leUser">
       <property name="Field" stdset="0">
        <string>f_user</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="5">
      <widget class="EQLineEdit" name="leGuest">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="6">
      <widget class="QGroupBox" name="grCityLedger">
       <property name="title">
        <string>City ledger</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="EQLineEdit" name="leCityLedgerCode">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Code</string>
          </property>
          <property name="Field" stdset="0">
           <string>f_cityLedger</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="EQLineEdit" name="leCityLedgerName">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="chUserFromBalance">
          <property name="text">
           <string>Transfer from balance</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="EQDoubleEdit" name="leCLbalance">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="EQLineEdit" name="lePaymentCode">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="Field" stdset="0">
        <string>f_paymentMode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="EQLineEdit" name="leState">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="Field" stdset="0">
        <string>f_state</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Guest</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbOperator">
       <property name="text">
        <string>Operator</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>AMD</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Balance</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Remarks</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>USD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbState">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item row="12" column="5">
      <widget class="EQLineEdit" name="leAmountUSD">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_amountUSD</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLineEdit" name="leTCVoucher"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqdoubleedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">eqdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">eqtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leReserveId</tabstop>
  <tabstop>lePaymentCode</tabstop>
  <tabstop>leCityLedgerCode</tabstop>
  <tabstop>leCardCode</tabstop>
  <tabstop>leAmount</tabstop>
  <tabstop>teRemarks</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnPrint</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>leAmountUSD</tabstop>
  <tabstop>leCityLedgerName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
